### ISSUE TITLE
Update repo URL to  arrow.apache.org/cookbook

### ISSUE THREAD

**Issue by thisisnic** [apache/arrow-cookbook#104]
Can we update the URL on the repo to point to  arrow.apache.org/cookbook instead of  arrow.apache.org/?  I'm not sure who can do this?

**Comment by westonpace** [comment]
Good catch.  Looks like you can do it here: https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features#Git.asf.yamlfeatures-GitHubsettings
